sequence_transform.c

```diff
@@ -224,6 +236,35 @@ void log_rule(st_trie_t *trie, st_trie_payload_t *res) {
     // Terminator
     uprintf("\n");
 }
+//////////////////////////////////////////////////////////////////////
+__attribute__((weak)) void st_on_missed_rule(const st_trie_rule_t *rule)
```

Comment:
In qmk weak functions which are meant for user are marked with `_user`, so I think we should use such notation too, to mark that this function is meant for user and not just some internal stuff

Reply:
Agreed. I also think that this function should use the full `sequence_transform_on_missed_rule` name to maintain consistency with `process_sequence_transform`, since `st_` doesn't mean as much to a regular user.

Reply:
Agreed.